docs/src/examples.rst

```diff
@@ -14,205 +14,265 @@
+  which creates the correct configuration to associate a cluster to AMI(s).
+  
+* Early validation of parameter combinations, which exposes |CFN| issues earlier, thus saving you deployment time.
+  For example, depending upon the task, it is easy to mis-configure the memory settings.
+  Previously you would not encounter an error until you deployed your app,
+  but now the |cdk| can detect a misconfiguration and emit an error when you synthesize your app.
 
-   code-block:: js
+* Automatically adds permissions for |ECR| if you use an image from |ECR|
+  When you use an image from |ECR|, the |cdk| adds the correct permissions.
 
-   Use **MyStack** and **DeploymentPipeline** to create the stacks and deployment pipeline.
+* Convenient API for autoscaling
+  The |cdk| supplies a method so you can autoscaling instances when you use an |EC2| cluster;
+  this functionality is done automatically when you use an instance in a Fargate cluster.
 
-   code-block:: js
+  In addition, the |cdk| will prevent instances from being deleted when
+  autoscaling tries to kill an instance,
+  but either a task is running or is scheduled on that instance.
 
-   import { App } from '@aws-cdk/cdk'
-   import { MyStack } from './my-stack'
-   import { DeploymentPipeline } from './my-deployment'
+  Previously, you had to create a Lambda function to have this functionality.
+  
+* Asset support, so that you can deploy source from your machine to |ECS| in one step
+  Previously, to use application source you had to perform a number of manual steps
+  (upload to |ECR|, create Docker image, etc.).
+ 
+.. _creating_ecs_l2_example_1:
 
-   const app = new App();
+Step 1: Create the Directory and Initialze the |cdk|
+----------------------------------------------------
 
-   // Use the default environment
-   new MyStack(app, { name: 'Dev' });
+Let's start with creating a new directory to hold our |cdk| code
+and create a new app in that directory.
 
-   // Pre-production stack
-   const preProd = new MyStack(app, {
-     name: 'PreProd',
-     env: { region: 'us-west-2' },
-     preProd: true
-   });
+.. code-block:: sh
 
-   // Production stacks
-   const prod = [
-     new MyStack(app, {
-       name: 'NAEast',
-       env: { region: 'us-east-1' }
-   }),
+    mkdir MyEcsConstruct
+    cd MyEcsConstruct
 
-   new MyStack(app, {
-     name: 'NAWest',
-     env: { region: 'us-west-2' }
-   }),
+.. tabs::
 
-   new MyStack(app, {
-     name: 'EU',
-     env: { region: 'eu-west-1' },
-       encryptedStorage: true
-     })
-   ]
+    .. group-tab:: TypeScript
 
-   // CI/CD pipeline stack
-   new DeploymentPipeline(app, {
-     env: { region: 'us-east-1' },
-      strategy: DeploymentStrategy.Waved,
-      preProdStages: [ preProd ],
-      prodStages: prod
-   });
+        .. code-block:: sh
 
-   app.run();
+            cdk init --language typescript
 
-.. _dynamodb_example:
+        Update *my_ecs_construct.ts* in the *bin* directory to only contain the following code:
```

Comment:
This will have gotten changed to `lib/my_ecs_construct_stack.ts`  by the time this goes out.

Reply:
I know. The change is not released yet. That's why I wrote:

> by the time this goes out.

Reply:
I removed this part as the template has changed.